cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -135,6 +136,49 @@ func (s *Server) Run() {
 	}
 }
 
+type IndexOptions struct {
+	// LargeFiles is a slice of glob patterns where matching files are
+	// indexed regardless of their size.
+	LargeFiles []string
+}
+
+func (o *IndexOptions) toArgs() string {
+	f := ""
+	for i := range o.LargeFiles {
+		f += "-large_file " + o.LargeFiles[i]
```

Comment:
you shoudl be building a `[]string` where arg is two elements of the string or a single string combined with `"-large_file=" + o.LargeFiles[i]`